src/classes/WalletManager/eip1193Provider.ts

```diff
@@ -3,15 +3,9 @@ export interface RequestArguments {
     params?: unknown[] | Record<string, unknown>
 }
 
-export interface MetaMaskEthereumProvider {
-    isMetaMask?: boolean
-
-    once(eventName: string | symbol, listener: (...args: any[]) => void): this
-
+export interface EIP1193Provider {
```

Comment:
shouldn't this extend ethers' `EIP1193Provider`?

Reply:
Ethers `Eip1193Provider` interface is not enough since doesn't have the event listener methods (`.on()`, `.addListener()`, etc...)